### PR TITLE
Add next player feature

### LINKED ISSUE #13: Implement a 'next player' feature to prevent miss hits

### PR DESCRIPTION
**Pull request by ggerbaud**
Fix #13

### CODE REVIEW

File: game/countup.go [hunk 1]
```diff
@@ -35,6 +35,11 @@ func NewGameCountUp(opt OptionCountUp) *CountUp {
 // HandleDart the implementation has to handle the Dart regarding the current player, the rules, and the context. Return a GameState
 func (game *CountUp) HandleDart(sector common.Sector) (result *common.GameState, error error) {
 
+	if game.state.Ongoing == common.ONHOLD {
```
Comment (mathieu-pousse) [inline game/countup.go]: Could we move that in a common method of Game ?

Comment (ggerbaud) [inline game/countup.go]: I guess it's more than just that part. I create an issue for that

**Comment by mathieu-pousse** [comment]
you should adapt the hardware/mock.go to disallow sending dart from sector "zero"

**Comment by mathieu-pousse** [comment]
LGTM

**Comment by erwannT** [comment]
LGTM

**Pull request merged**